watchtower/wtclient/client.go

```diff
@@ -545,10 +570,72 @@ func (c *TowerClient) Start() error {
 			}
 		}
 
+		chanSub, err := c.cfg.SubscribeChannelEvents()
+		if err != nil {
+			returnErr = err
+			return
+		}
+
+		// Iterate over the list of registered channels and check if
+		// any of them can be marked as closed.
+		for id := range c.summaries {
```

Comment:
The code initializing the summaries uses `cfg.DB.FetchChanSummaries()`, which does not return closed summaries. On my node, I was able to delete ~20% of all sessions. I don't know why it stopped marking other sessions as deletable, but I assume that I'd suffice to re-evaluate based on the existing code and `c.summaries` including closed channels (!).